### PR TITLE
Add support for analytical ttl to java sdk

### PR DESCRIPTION
**Pull request by moderakh**
Added support for analytical ttl:

sample usage
```java
CosmosAsyncDatabaseResponse database = client
    .createDatabaseIfNotExists("testDB")
    .block();

CosmosContainerProperties cosmosContainerProperties = 
    new CosmosContainerProperties("testCol", "/id");
cosmosContainerProperties.setAnalyticalStorageTimeToLiveInSeconds(-1);

database.getDatabase().createContainer(cosmosContainerProperties).block();
```

fixed a bug in api `CosmosContainerProperties#setDefaultTimeToLiveInSeconds()` should return `CosmosContainerProperties`

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java [hunk 1]
```diff
@@ -211,33 +211,76 @@ public Integer getDefaultTimeToLiveInSeconds() {
...
      * <p>
-     * -1 - indicates by default all documents within the collection never expire. This value can be overridden by 
+     * -1 - indicates by default all documents within the collection never expire. This value can be overridden by
      * individual documents'
      * time-to-live value.
      *
      * @param timeToLive the default time-to-live value in seconds.
      */
-    public void setDefaultTimeToLiveInSeconds(Integer timeToLive) {
+    public CosmosContainerProperties setDefaultTimeToLiveInSeconds(Integer timeToLive) {
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java:233]: Nice catch, are there any such left overs?

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AnalyticalStorageCodeSnippet.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.azure.cosmos;
```
Comment (mbhaskar) [inline sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AnalyticalStorageCodeSnippet.java:1]: Missing license header.

Comment (moderakh) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AnalyticalStorageCodeSnippet.java:1]: good catch, thanks will fix.

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java (hunk 1 above)
Comment (moderakh) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java:233]: seems there are a few other places as well.

@kushagraThapar  FYI
```bash
find . | egrep "Properties|Setting|Config|Option" | grep -v implementation| xargs grep void
./java/com/azure/cosmos/models/CosmosStoredProcedureProperties.java:    public void setBody(String body) {
./java/com/azure/cosmos/models/FeedOptions.java:     * was done but not yet utilized. This avoids redoing the work again in
./java/com/azure/cosmos/models/FeedOptions.java:    public void setEmptyPagesAllowed(boolean emptyPagesAllowed) {
```

Comment (moderakh) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java:234]: ok I fixed all of them in this PR.

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AnalyticalStorageCodeSnippet.java (hunk 2 above)
Comment (moderakh) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/AnalyticalStorageCodeSnippet.java:4]: addressed. thanks.

**Pull request merged**